### PR TITLE
Update to r118.1

### PR DESCRIPTION
**Pull request by I-Iomer**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     <groupId>org.webjars</groupId>
     <artifactId>three.js</artifactId>
-    <version>r89-SNAPSHOT</version>
+    <version>r118.1</version>
```
Comment (jamesward) [inline pom.xml:13]:
```suggestion
    <version>r118.1-SNAPSHOT</version>
```

**Comment by jamesward** [comment]
I don't see the tag `r118.1` at https://github.com/mrdoob/three.js/tags
How about `r118` instead?

**Comment by I-Iomer** [comment]
Yes, sure!
Thank you!

**Comment by jamesward** [comment]
`r118` is being released shortly.

**Comment by I-Iomer** [comment]
Hi!
There is additional file in a build from master:
three.module.js

May you add it also to jar?

Jar should contain:
three.js
three.min.js
three.module.js

Thank you in advance!

**Comment by jamesward** [comment]
I'm releasing `r118-1` with this shortly.

**Comment by I-Iomer** [comment]
Great! Thank you!

**Comment by I-Iomer** [comment]
Hi, James!
I need your help. Three.js have additional extensions (modules) for example: OrbitControls, Stats and etc.
all of them stored here:
https://github.com/mrdoob/three.js/tree/dev/examples/jsm

Is it possible to add them all to jar or maybe it is better to create separated three.js modules package?
Thank you in advance!

**Comment by jamesward** [comment]
Here is another option... I'm deploying the latest NPM WebJar for `three` now:
https://www.npmjs.com/package/three

This is a direct copy of the NPM module so it has everything in it.  And you can deploy future new versions from https://webjars.org

Think that'll work?

**Comment by I-Iomer** [comment]
OK, lets try!

**Comment by jamesward** [comment]
Let me know how it goes.

**Comment by I-Iomer** [comment]
Hi! Maybe I'm wrong...
I tried to make build yesterday on https://www.webjars.org/ through Add a WebJar
NPM -> three -> 0.118.3 -> was done
today did the same log:
Starting Deploy
Got package info for org.webjars.npm three 0.118.3
Determining dependency graph
No dependencies.
WebJar org.webjars.npm three 0.118.3 has already been deployed

but on maven repos:
https://mvnrepository.com/artifact/org.webjars.npm/three.js

still nothing, 0.77.1 is last

here (https://mvnrepository.com/artifact/org.webjars/three.js) only yours build r118-1

**Comment by jamesward** [comment]
The artifact name is `three` not `three.js` to match the NPM package name.

**Comment by I-Iomer** [comment]
Sorry, my mistake
Everything workin fine, thank you!

Спасибо! =)

**Pull request merged**